<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>257</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>168</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="textLabel">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Witch Button Click?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="leftButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="rightButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
